spec/ruby/core/module/using_spec.rb

```diff
@@ -243,6 +243,96 @@ def self.call_foo(c)
       mod.call_foo(c).should == "foo from refinement"
     end
 
+    it "is active for module defined via initializer" do
```

Comment:
I think `via initializer` is a bit unclear, I would suggest `via Module.new {}` / `via Class.new {}`